src/app/dataset-view/additional-components/metadata-component/components/source-events/add-polling-source/add-polling-source.component.html

```diff
@@ -1,6 +1,7 @@
 <div class="layout" *ngIf="history">
     <div class="mt-4">
         <div class="border border-1 mt-4 px-2 py-4 rounded-3">
+            <pre>{{ pollingSourceForm.value | json }}</pre>
```

Comment:
Looks like debug code that is easy to miss

Reply:
Removed